charts/camunda-platform/Chart.yaml

```diff
@@ -1,41 +1,59 @@
 apiVersion: v2
 name: camunda-platform
-version: 8.0.11
+version: 8.0.12
```

Comment:
:thought_balloon: [Did you use the release script](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/release.sh)? :) Just want to make you aware that this exist.

Reply:
I actually used the script :grin: 
it makes the life easier :100:

Reply:
Nice :+1: